In [1]:
import pandas as pd

# Read the CSV file into a DataFrame
df = pd.read_csv("Question 3.csv")

# Display the first few rows of the DataFrame to inspect its content
df.head()


Comments  Ratings
0  I didnt go in with big hopes, but i was expect...        8
1  A unique genre, a well written story (script) ...        8
2  Majestic at scale, grandeur in VFX, and great ...        9
3  "Kalki 2898" is not just a movie; it's an expe...       10
4  Best Indian movie Nagi combined Hindu mytholog...       10

In [2]:
from textblob import TextBlob

# Review text provided by the user
review_text = ("'Queen of Katwe' is not just a chess movie; it's a celebration of "
               "the human spirit and the incredible potential that lies within us all. "
               "It's an inspiring and uplifting tale that will leave you feeling both "
               "moved and motivated. This is a must-see film for anyone who values "
               "stories of triumph against adversity and the pursuit of one's dreams.")

# Perform sentiment analysis using TextBlob
analysis = TextBlob(review_text)
sentiment_score = analysis.sentiment.polarity

# Display the sentiment score
sentiment_score


0.35

The sentiment score for the review is 0.35.

Interpretation:
Sentiment Score Range:
-1 indicates very negative sentiment.
0 indicates neutral sentiment.
+1 indicates very positive sentiment.
A score of 0.35 suggests that the review has a moderately positive sentiment, indicating a generally favorable opinion about the movie "Queen of Katwe."

Corresponding Rating:
Given the positive nature of the sentiment, this review would likely correspond to a high rating. Based on typical sentiment-rating mappings, this might translate to a rating in the range of 7 to 9 out of 10.